debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -148,6 +148,14 @@ public static Version parse(String value, String defaultValue) {
             .withDescription("A version of the format of the publicly visible source part in the message")
             .withValidation(Field::isClassName);
 
+    public static final Field SANITIZE_FIELD_NAMES = Field.create("sanitize.field.names")
+            .withDisplayName("Sanitize field names to adhere to AVRO naming conventions")
```

Comment:
Should be Avro instead of AVRO; I've pushed a commit to master for changing it.

Reply:
@gunnarmorling Thanks!